tests/misc_env/install_prereq.py

```diff
@@ -173,6 +184,41 @@ def enable_rhel_rpms(ceph, distro_ver):
         )
 
 
+def enable_rhel_eus_rpms(ceph, distro_ver):
+    """
+    Setup cdn repositories for rhel systems
+    Args:
+        distro_ver: distro version details
+        ceph: ceph object
+    """
+
+    eus_repo = {"7": ("7.7", ["rhel-7-server-eus-rpms"])}
+
+    ceph.exec_command(
+        sudo=True,
+        cmd="subscription-manager release --set={distro}".format(
+            distro=eus_repo.get(distro_ver[0])[0]
```

Comment:
If we hit an `IndexError` here, or a `TypeError` on `None`, it will be very hard to track down the bug, because we're doing so much in this one line. Would you please reformat this code so the assignments and backtraces are easier to understand?

Here's a suggestion:

```python
rhel_major_version = distro_ver[0]  # eg. "7"
if rhel_major_version == "7":
    # We only support one EUS release for RHEL 7:
    release = "7.7"
else:
    raise NotImplementedError("cannot set EUS repos for %s", rhel_major_version)
cmd = f"subscription-manager release --set={release}"

Reply:
same reply as above. let me look into it

Reply:
done.